src/utils/clamp.ts

```diff
@@ -0,0 +1,5 @@
+// Clamps a value between an upper and lower bound.
+// Used ternary operator instead of `Math.min(Math.max(min, number), max)` to save few bytes
+export const clamp = (number: number, min = 0, max = 1): number => {
+  return number > max ? max : number < min ? min : number;
```

Comment:
```suggestion
  return Math.max(min, Math.min(number, max));
```

FWIW, I think this is like ~10-20b smaller. Counting pennies, but personally I think it's more readable too.

Reply:
In this project, the output size is more important than the source code style. 
We use tricks and they make code worse-looking but we cover then with comments.

Reply:
Yes, what I'm saying is that the suggestion I provided is 10-20b lighter. 

[Existing outputs to 207 bytes](https://babeljs.io/repl#?browsers=IE%2011&build=&builtIns=usage&spec=false&loose=true&code_lz=MYewdgzgLgBMA2BDAtgBxgXhgCjAV2QCMBTAJwBoZkBLMTGABkuUQA96BGASkwD4YA3gCgYMUsSh5SdALKIoACwB0LVthphKcxStq4CJClTZcuAbiEBfIUA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=true&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Cenv&prettier=false&targets=&version=7.11.6&externalPlugins=)

[My suggestion outputs to 195](https://babeljs.io/repl#?browsers=IE%2011&build=&builtIns=usage&spec=false&loose=true&code_lz=MYewdgzgLgBMA2BDAtgBxgXhgCjAV2QCMBTAJwBoZkBLMTGABkuUQA96BGASkwD4YA3gCgYMUsSh5SdALKIoACwB0LVthphKcxStq4CJClTZcuAbiEBfIUA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=true&timeTravel=false&sourceType=module&lineWrap=false&presets=env%2Cenv&prettier=false&targets=&version=7.11.6&externalPlugins=)

Reply:
So you shave off 12b using `Math.max()` and the like

Reply:
Got you. But you should compare minified code. Solution based on ternary operators is almost 2 times shorter.
```js
Math.max(i,Math.min(n,a));
n>a?a:n<i?i:n;
```

Reply:
Ah, true enough.

Reply:
Described this thing in 6e810e6

Reply:
Cheers. Sorry about that.